client/src/components/ratings&reviews/ReviewTile.jsx

```diff
@@ -8,7 +10,7 @@ class ReviewTile extends React.Component{
   render() {
     return (
       <div>
-        <h3> Rating: 0/5 </h3>
+        <div className="stars">{stars(3.75)}</div>
```

Comment:
Is the className of "stars" essential for how it renders appropriately?  If so, this might make more sense to include inside the component itself as something that comes as a full package?

Reply:
You're right. I deleted the classname and it was fine. I had initially put it in there because the stars were not rendering side by side but on separate lines. However, in the .css file, the default setting for 'img' was what was actually affecting it. I will change and update.